src/tests/fetchAPI.test.js

```diff
@@ -0,0 +1,13 @@
+import fetchCoins from './__mocks__/fetchCoins';
+
+describe('Genres must pass the test', () => {
+  test('Fetch available Coins must return data', () => {
+    expect(fetchCoins()).toBeDefined();
+  });
+  test('Fetch Coins return value length must be 5', () => {
+    expect(fetchCoins()).toHaveLength(3);
+  });
+  test('Fourth fetched genre name should be Award Winning', () => {
+    expect(fetchCoins()[2].name).toBe('XRP');
+  });
+});
```

Comment:
- [ ] Hello, Well done in implementing most of the changes. The previous review asked the test should be implemented based on the project requirements available. Here below what you need to do:

1. Your app must have unit tests for several components of the app and some pure functions in Redux code.
2. Your app must have integration tests using React Testing Library
   * you can perform a mocking data fetching and Redux Store connection.